doc/README.yaml.md

```diff
@@ -0,0 +1,240 @@
+
+# README for Open-CE build YAML files
+
+
+This README documents the YAML files used by the build scripts
+[build_env.py](https://github.com/open-ce/open-ce/blob/master/doc/README.build_env.md)
+and [build_feedstock.py](https://github.com/open-ce/open-ce/blob/master/doc/README.build_feedstock.md).
+Please refer to the respective README files for each of these two scripts.
+
+Most likely you will want to simply use the YAML files provided within the
+open-ce repository as they are. These files have already been customized and
+verified to work. However, if you want to debug or do your own customizations,
+you can edit them or create your own YAML files for use by these scripts. If you
+do this, you will likely want to start by copying one of the existing files and
+making local edits as desired.
+
+NOTE: This file only documents information about the YAML files as used within
```

Comment:
This NOTE is not needed.

Reply:
Removed the NOTE. Thought it might be helpful to point to external official YAML documentation.... but then again if someone is customizing these files they probably know how to find such info if they need.